addons/auth_signup/views/auth_signup_login_templates.xml

```diff
@@ -103,4 +103,88 @@
+                    <td align="center" style="min-width: 590px;">
+                        <table border="0" cellpadding="0" cellspacing="0" width="590" style="min-width: 590px; background-color: white; padding: 0px 8px 0px 8px; border-collapse:separate;">
+                            <tr><td valign="middle">
+                                <span style="font-size: 10px;">Your Account</span><br/>
+                                <span style="font-size: 20px; font-weight: bold;">
+                                    <t t-out="object.name or ''">Marc Demo</t>
+                                </span>
+                            </td><td valign="middle" align="right" t-if="not object.company_id.uses_default_logo">
+                                <img t-attf-src="/logo.png?company={{ object.company_id.id }}" style="padding: 0px; margin: 0px; height: auto; width: 80px;" t-att-alt="object.company_id.name"/>
+                            </td></tr>
+                            <tr><td colspan="2" style="text-align:center;">
+                              <hr width="100%" style="background-color:rgb(204,204,204);border:medium none;clear:both;display:block;font-size:0px;min-height:1px;line-height:0; margin: 16px 0px 16px 0px;"/>
+                            </td></tr>
+                        </table>
+                    </td>
+                </tr>
+                <!-- CONTENT -->
+                <tr>
+                    <td align="center" style="min-width: 590px;">
+                        <table border="0" cellpadding="0" cellspacing="0" width="590" style="min-width: 590px; background-color: white; padding: 0px 8px 0px 8px; border-collapse:separate;">
+                            <tr><td valign="top" style="font-size: 13px;">
+                                <div>
+                                    Dear <t t-out="object.name or ''">Marc Demo</t>,<br/><br/>
+                                    A password reset was requested for the Odoo account linked to this email.
+                                    You may change your password by following this link which will remain valid during 24 hours:<br/>
+                                    <div style="margin: 16px 0px 16px 0px;">
+                                        <a t-att-href="object.signup_url"
+                                            style="background-color: #875A7B; padding: 8px 16px 8px 16px; text-decoration: none; color: #fff; border-radius: 5px; font-size:13px;">
+                                            Change password
+                                        </a>
+                                    </div>
+                                    If you do not expect this, you can safely ignore this email.<br/><br/>
+                                    Thanks,
+                                    <t t-if="user.signature">
+                                        <br/>
+                                        <t t-out="user.signature or ''">--<br/>Mitchell Admin</t>
```

Comment:
Do you need the empty string in this `t-out` since you checked beforehand that user.signature exists and that an empty string is falsy ? Or did I miss something here ?